### PR TITLE
Fix web app and add NPM scripts

### PR DESCRIPTION
**Pull request by Meakk**

### CODE REVIEW

File: webassembly/README.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+F3D is a fast and minimalist 3D viewer. Implemented in C++, it has been cross-compiled with emscripten to a WebAssembly module.
```
Comment (Meakk) [inline webassembly/README.md:1]: This readme file is the one that will be displayed here: https://www.npmjs.com/package/f3d

Comment (mwestphal) [reply webassembly/README.md:1]: nice

Comment (mwestphal) [reply webassembly/README.md:1]: I dont see anything to publish the npm, i suppose it comes in another PR ?

**Review by mwestphal — APPROVED** [review]

File: webassembly/README.md (hunk 1 above)
Comment (Meakk) [reply webassembly/README.md:1]: Yes indeed. Publishing will come later.  
It's possible to do it manually with `npm publish` though.

Comment (mwestphal) [reply webassembly/README.md:1]: noted

**Pull request merged**